src/raghvendra/StringAssignments/PrintWords.java

```diff
@@ -0,0 +1,32 @@
+package raghvendra.StringAssignments;
+/* Program 1: print all the words which does't have any digit. 
+String[] input = {"Masgj3Pw", "r4fsjk","techno","sfjk44lk"};
+output : techno*/
+public class PrintWords {
+	void printWordwithoutDigit(String[] st) {
+		int count;
+		System.out.println("Words without digit is: ");
+		for(int index=0;index<st.length;index++) {
+			String word=st[index];
+			count=0;
+			for(int inner=0;inner<word.length();inner++) {
+				char ch=word.charAt(inner);
+				if (Character.isDigit(ch)) {
+					count++;
```

Comment:
Looks like you have good hand on counter, now you should start using boolean values to handle this conditional part